### DISCUSSION TITLE
Elderly women surpassed youth in numbers in the Netherlands, but men did not (yet)

### DISCUSSION THREAD

**Discussion by xomnia-bobdewitte** [has2k1/plotnine#833] [2024 Plotnine Contest]
### Authors

Bob de Witte

### Full description

In this submission, an aging population in the Netherlands is explored, utilizing a dataset from the Dutch Central Bureau of Statistics (CBS) which provides a detailed age and gender breakdown. The data from CBS's Population Pyramid visualizes shifts in the population structure, offering a profound insight into the demographic changes over recent years.

### Submission plot
![submission_plot](https://github.com/user-attachments/assets/c9d3f59e-8b1a-4a04-bdff-cf5b9e66a5d7)

### Code repository

https://github.com/xomnia-bobdewitte/plotnine-contest-NLAgeGroups

### Notebook link

https://github.com/xomnia-bobdewitte/plotnine-contest-NLAgeGroups/blob/master/notebook.ipynb

**Comment by NatashaData** [comment]
Hi Bob,
Nice work, this is really interesting and a handy visualization!
I'm just wondering, the half of the graphs from 2025 onwards, is this also from the data source (as projected trajectory estimates for the future) or did you just extend the plot based on curves of best fit? I'm wondering if solid lines to the left of 2025 showing actual data, transitioning to dotted lines post 2025 may show this distinction more clearly as well.
Well done!

**Reply by xomnia-bobdewitte** [reply]
Hi Natasha, 
Thanks for your interest in the visualization!
Data after 2024 (2025 onwards) is forecasted bij [CBS](https://www.cbs.nl/nl-nl/visualisaties/dashboard-bevolking/bevolkingspiramide) , so it is indeed coming from the data source! It's actually a nice suggesting to have forecasted data in dotted line form for instance, and historic data as a solid line! Thanks :)
Cheers,
Bob

**Comment by has2k1** [comment]
![download](https://github.com/user-attachments/assets/3f4222db-8d3d-4b0d-bdef-bdd5eb51ec5c)

<hr/>

![download (1)](https://github.com/user-attachments/assets/59ce064a-91f1-448f-991a-34914e003ad3)

**Comment by EconomiCurtis** [comment]
Ahoy @xomnia-bobdewitte , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Comment by has2k1** [comment]
@xomnia-bobdewitte, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
